db/seeds/fincap_data/redirects.rb

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    source: '/document/WOuEzCsAAIaQ8tXy/financial-well-being-the-employee-view',
+    destination: '/en/insights/financial-well-being-the-employee-view'
+  },
+  {
+    source: '/document/WTVdRSkAAOsYaAQx/raising-household-saving',
+    destination: '/en/reviews/raising-household-saving'
+  },
+  {
+    source: '/document/WOYJaysAADA42rIr/looking-after-the-pennies',
+    destination: '/en/evaluations/looking-after-the-pennies'
+  },
+  {
+    source: '/young-adults',
+    destination: '/en/lifestages/young-adults',
+  },
+  {
+    source: '/en/news/old-news',
+    destination: '/en/news/press-release-a-new-way-to-pay'
+  }
+].each do |attributes|
+  Redirect.create!(attributes.merge(redirect_type: 'permanent'))
```

Comment:
I don't think we should bang this, seeds should be run idempotently, as many times as you want, right?

Reply:
Yes, that's true. I updated using `find_or_create-by`. ♻️